proxy/src/main/java/net/md_5/bungee/scheduler/BungeeTask.java

```diff
@@ -83,4 +89,13 @@ public void run()
 
         cancel();
     }
+
+    private static LogRecord log(Throwable t, String msg, Object[] parameters, Logger logger)
```

Comment:
What's the purpose of this change?

Reply:
Have parameters and an exception in a logrecord. For some reason its missinng in the log overloads

Reply:
Just concat it, or yet leave it - how many times a second do you realistically expect a plugin to throw an exception in a task?

Reply:
I don't see the problem in adding this method. Code is clear.

Reply:
It's pointless code though.
If you were writing this code from scratch it's highly unlikely you'd make a method like that

Reply:
In that case, edit it yourself please how you would like to have it (I have allow edits from maintainers on).